model/model.mustache

```diff
@@ -73,25 +74,30 @@ module.exports = {
     if (!{{name}}.rev) {
       return Promise.reject(new httpError('BAD_REQUEST', 'No rev provided'));
     }
-    const valid = schema.validate({{name}});
-    if (!valid) throw new httpError('UNPROCESSABLE_ENTITY');
-    return r.table('{{pluralName}}')
-    .get({{name}}.id)
-    .update(
-      rethinkdb.ifRevMatches({{name}}),
-      { returnChanges: true }
-    )
-    .run()
-    .then(rethinkdb.checkRevError)
-    .then(rethinkdb.firstChange);
+
+    return r.table('{{pluralName}}').get({{name}}.id).run()
+    .then(current => {
+      const updated = Object.assign({}, current, {{name}});
```

Comment:
Fair point, that was my other train of thought. I'll update this when I get time. One slight complexity is the user model though and always having to send across a password property since it's required. That feels kinda gross might need to do a workaround.